pyperf/_runner.py

```diff
@@ -201,6 +201,9 @@ def __init__(self, values=None, warmups=None, processes=None,
                             help='Comma-separated list of environment '
                                  'variables inherited by worker child '
                                  'processes.')
+        parser.add_argument("--copy-env-vars",
```

Comment:
I'm lazy and I would prefer to just type: --copy-env. Maybe --copy-environ, since argparse allows to only type "--copy-env", since there is no conflict ;-)

Reply:
@vstinner Done! ;)